lib/widgets/login.dart

```diff
@@ -0,0 +1,188 @@
+                  loginContainersGeneral(context, 'Email ', 'General'),
+                  loginContainersGeneral(context, 'Senha ', 'Password'),
+                  loginContainersGeneral(
+                      context, 'Confirmar senha ', 'Password'),
+                ],
+              ),
+            ),
+          ),
+          Container(
+            child: Container(
+              width: screenSize(context, "width", 1),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                children: [
+                  smallContainer(context, 'Voltar', 0XFFE57373),
+                  smallContainer(context, 'Submeter', 0XFF8BC34A)
+                ],
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget insertText(BuildContext context, String text, String typeContainer) {
+    bool key = false;
+
+    if (typeContainer == 'Password') key = true;
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        TextField(
+          style: TextStyle(color: Colors.white),
+          decoration: InputDecoration(
+              labelText: "  " + text,
+              labelStyle: TextStyle(
+                  color: white,
+                  fontSize: screenSize(context, "width", 0.048),
+                  fontFamily: 'DMSans'),
+              filled: true,
+              fillColor: Colors.black,
+              border: OutlineInputBorder()),
+          obscureText: key,
+        )
+      ],
+    );
+  }
+
+  Widget loginContainersName(
+      BuildContext context, String text, String typeContainer) {
+    return Padding(
+      padding: EdgeInsets.symmetric(
+          horizontal: screenSize(context, "width", 0.012),
+          vertical: screenSize(context, "height", 0.003)),
+      child: Container(
+        width: screenSize(context, "width", 0.44),
+        height: screenSize(context, "height", 0.1),
+        child: insertText(context, text, typeContainer),
+      ),
+    );
+  }
+
+  Widget loginContainersGeneral(
+      BuildContext context, String text, String typeContainer) {
+    return Padding(
+      padding: EdgeInsets.all(screenSize(context, "height", 0.003)),
+      child: Container(
+        width: screenSize(context, "width", 0.9),
+        height: screenSize(context, "height", 0.1),
+        child: insertText(context, text, typeContainer),
+      ),
+    );
+  }
+
+  Widget imageCircle(BuildContext context) {
+    double userRadius = screenSize(context, "height", 0.179);
+    double imageRadius = screenSize(context, "height", 0.0704);
+
+    return Stack(
+      children: <Widget>[
+        Container(
+            height: userRadius,
+            width: userRadius,
+            decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(userRadius), color: white),
+            child: Center(
+                child:
+                    Image(image: AssetImage('assets/images/user_figure.png')))),
```

Comment:
O ideal aqui é utilizar o ``` Icon ``` do flutter, utilizei o seguinte
```suggestion
                    Icon(
                Icons.person_outlined,
                size: userRadius * 0.65,
              ),
```